index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="portfolio.css">
+</head>
+<body>
+    <div class="page-container">
+
+    <header>
+        <nav>
+            <ul class="header-container">
+                <li class="Nav Nav-1">
+                  <a href="">Welcome</a>
+                </li>
+
+                  
+                <li class="Nav Nav-2">
+                    <a href=""><img src="images/Menu.svg" alt="a Union icon" class="social-icons"></a>
+                </li>
+            </ul>
+        </nav>
+        
+    </header>
+
+    <section id="s-1">
+        <div class="section-title">
+            <h1>
+                Hey there. I’m Mariane
+                <br>
+               <span>I’m a software developer</span> 
+            </h1>
+            
+        </div>
+        
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+
+        <nav>
+            <ul>
+                <li>
+                  <a href="">
+                      <img src="images/twitter.svg" alt="" class="social-icons">
+                  </a>
+                </li>
+
+                <li>
+                   <a href="">
+                    <img src="images/Linkedin-icon.svg" alt="" class="social-icons">
+                   </a>
+                </li>
+
+                <li>
+                    <a href="">
+                        <img src="images/medium.svg" alt="" class="social-icons">
+                    </a>
+                </li>
+
+                <li>
+                    <a href="">
+                        <img src="images/github.svg" alt="" class="social-icons">
```

Comment:
- [ ] This is not a `nav` tag it should be `div class= "social-media"`, Please fix this